latest_storie_css.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html>
+    <head>
+       <!-- <meta name="viewport" content="width=deive-width, initial-scale=1.0"> -->
+        <style>
+            
+            body{
+                display:block;
+                margin: 1px;
+                
+            }
+            body > div{
+                padding-inline-start: 1px;
+            }
+            .grid-container{
+                display: grid;
+                grid-template-columns: auto auto auto auto auto auto auto;
+                grid-gap: 0px;
+                margin: 0px 0px 0px 0px;
+                padding-inline-start: 1px;
+                
+            }
+            .grid-container > div{
+                padding-bottom: 10px;
+                
+            }
+            .verticle{
+                border-right: 1px solid grey;
+                height: 280px;
+               
+                width: 300px;
+                
+            }
+        </style>
+    </head>
+    <body>
+        <h1>Latest stories</h1>
+
+        <!-- storie images-->
+    <div class="grid-container">    
+        <div class="verticle" >
+            <article>
+                <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRdg-3vrifQILpxjwuytFnp5xtVPsNhQzqGh5H69yRsJlEGaiUU&s" alt="nature" width="290" heigth="250"/>
+                <figcaption style="font-size: 12px; color: grey;"> December 11, 2017</figcaption></br>
```

Comment:
Ideally a figcaption comes after a figure tag.  In this case there is no figure tag.